src/main/java/bg/ehealth/prescriptions/persistence/model/PharmacyView.java

```diff
@@ -0,0 +1,8 @@
+package bg.ehealth.prescriptions.persistence.model;
+
+public interface PharmacyView {
```

Comment:
I think we could do without projections in general. It's fine to stay here if you think, though.